### ISSUE TITLE
NTR: Cardiac imaging: left ventricle aneurysms (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9029]
New term request Cardiac imaging: left ventricle aneurysms (nadea3_190419143827): Echocardiogram  UMLS: 2199707

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000144
- label: Left ventricular aneurysm
- parents: [Abnormal left ventricle morphology (HP:0001711)](https://hpo.jax.org/app/browse/term/HP:0001711)
- pmid: PMID:32310415
- def: A severely scarred portion of infarcted ventricular myocardium that does not contract normally but instead bulges outward while the rest of the ventricle contracts during systole.
- comment: A left ventricular aneurysm is one of the worst complications after acute myocardial infarction. The natural course leading to the formation of a ventricular aneurysm involves a full-thickness infarct that has been replaced by fibrous tissue. This inert portion cannot take part in the contraction and herniates outward during systole. Most of the LV aneurysms are asymptomatic and are found in follow-up echocardiography. However, large symptomatic LV aneurysms can cause heart failure, ventricular tachyarrhythmias, and sudden cardiac death.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9029
- synonyms:
